app/models/resume.rb

```diff
@@ -0,0 +1,23 @@
+class Resume < ApplicationRecord
+  #params.require(:resume).permit(:title, :revision, :user_name, :user_id)
+  has_one_attached :data
+  validates :title, presence: true
+  validates :revision, presence: true
+  validates :user_id, presence: true
+
+  attr_accessor :resume_data
+
+  after_create :parse_doc
+
+  def parse_doc
```

Comment:
This method is being called when creating a new resume object in order to parse the base64 to a pdf file.